CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2020-02-10 - Fix gemspec versions
```

Comment:
The changelog needs to include the release version

Reply:
@op-ct I just matched the other ones but can do